website/content/docs/connect/gateways/mesh-gateway/service-to-service-traffic-peers.mdx

```diff
@@ -0,0 +1,54 @@
+---
+layout: docs
+page_title: Service-to-service Traffic Across Peered Clusters
+description: >-
+  This topic describes how to configure mesh gateways to route a service's data to upstreams
+  in clusters that have a peering connection.
+---
+
+# Service-to-service Traffic Across Peered Clusters
+
+~> **Cluster peering is currently in beta**: Functionality associated with cluster peering is subject to change. You should never use the beta release in secure environments or production scenarios. Features in beta may have performance issues, scaling issues, and limited support.
+
+Mesh gateways are required for you to route service mesh traffic between different Consul clusters. Clusters can reside in different clouds or runtime environments where general interconnectivity between all services in all clusters is not feasible.
+
+Unlike mesh gateways for datacenters and partitions, mesh gateways for cluster peering decrypts data to HTTP services within the mTLS session. Data must be decrypted in order to apply dynamic routing rules configured in the destination cluster.
```

Comment:
This point about decryption may be contentious, so It would probably help to expand a bit more on why dynamic routing rules are evaluated at mesh gateways.

Maybe something like this could be added at the end:

> Dynamic routing rules are evaluated at destination clusters to reduce coupling between peers.